src/Pages/Airline/MainSearch/MainSearch.jsx

```diff
@@ -0,0 +1,282 @@
+import React from "react";
+import { Link, withRouter } from "react-router-dom";
+import AirCalendar from "../AirlineCalendar/AirCalendar";
+import CityModal from "../Modal/CityModal";
+import SeatModal from "../Modal/SeatModal";
+import moment from "moment";
+import "moment/locale/ko";
+import styled from "styled-components";
+
+class MainSearch extends React.Component {
+  constructor() {
+    super();
+    moment.locale("ko");
+    this.state = {
+      openCityModal: false,
+      openSeatModal: false,
+      openCalendarLayer: false,
+      cityValue: "",
+      countA: 1,
+      countB: 0,
+      countC: 0,
+      passengerNum: 0,
+    };
+  }
+
+  handleCityValue = (e) => {
+    this.setState({
+      cityValue: e.target.innerText,
+    });
+    this.showCityModal();
+  };
+
+  showCityModal = () => {
+    const { openCityModal } = this.state;
+    this.setState({
+      openCityModal: !openCityModal,
+    });
+  };
+
+  showSeatModal = () => {
+    const { openSeatModal } = this.state;
+    this.setState({
+      openSeatModal: !openSeatModal,
+    });
+  };
+
+  showCalendarLayer = () => {
+    const { openCalendarLayer } = this.state;
+    this.setState({
+      openCalendarLayer: !openCalendarLayer,
+    });
+  };
+
+  DecreaseA = () => {
+    const { countA } = this.state;
+    if (countA > 1) {
+      this.setState({
+        countA: countA - 1,
+      });
+    }
+  };
+
+  IncreaseA = () => {
+    const { countA, countB, countC } = this.state;
+    if (countA + countB + countC < 9) {
+      this.setState({
+        countA: countA + 1,
+      });
+    } else {
+      alert("최대 9명까지만 예약 가능합니다.");
+    }
```

Comment:
단일 if, else문이네요! 무엇을 사용하면 **더욱 가독성 좋은 깔끔한 코드**가 될까요?
더불어 if문 안에 조건이 길어질 경우 변수에 선언하여 사용 할 수 있습니다.

```suggestion
    const increaseCondition = countA + countB + countC < 9
    if (increaseCondition) {
      this.setState({
        countA: countA + 1,
      });
    } else {
      alert("최대 9명까지만 예약 가능합니다.");
    }
```